scripts/copy-mod-doc-exceptions.txt

```diff
@@ -1159,13 +1159,6 @@ src/linear_algebra/basic.lean : line 2054 : ERR_LIN : Line has more than 100 cha
 src/linear_algebra/basic.lean : line 2112 : ERR_LIN : Line has more than 100 characters
 src/linear_algebra/basic.lean : line 2180 : ERR_LIN : Line has more than 100 characters
 src/linear_algebra/basis.lean : line 93 : ERR_LIN : Line has more than 100 characters
-src/linear_algebra/bilinear_form.lean : line 106 : ERR_LIN : Line has more than 100 characters
```

Comment:
While it's not necessary to edit this manually, I suppose it's a good check that you've indeed cleared the long lines from the file and I don't think it will cause any problems.